CONTRIBUTING.md

```diff
@@ -36,44 +43,70 @@ The following is a summary of the ideal contribution flow. Please, note that Pul
     └───────────────────────┘
 ```
 
-## Code of Conduct
-AsyncAPI has adopted a Code of Conduct that we expect project participants to adhere to. Please [read the full text](./CODE_OF_CONDUCT.md) so that you can understand what sort of behaviour is expected.
+Issues and pull requests without activity from the creator within 14 days will be automatically closed by a triager or committer. However, closure does not mean rejection. If you wish to revisit a closed issue or pull request, feel free to open a new one referencing the closed item.
 
-## Our Development Process
-We use Github to host code, to track issues and feature requests, as well as accept pull requests.
+Issues and pull requests without activity from the triager or committer within 14 days may occure for many different reasons. Creator may use `/ptal` comment in the pull request to call out maintainers.
+
+### Issues
 
-## Issues
-[Open an issue](https://github.com/asyncapi/asyncapi/issues/new) **only** if you want to report a bug or a feature. Don't open issues for questions or support, instead join our [Slack workspace](https://www.asyncapi.com/slack-invite) and ask there. Don't forget to follow our [Slack Etiquette](https://github.com/asyncapi/community/blob/master/slack-etiquette.md) while interacting with community members! It's more likely you'll get help, and much faster!
+[Open an issue](https://github.com/asyncapi/asyncapi/issues/new) **only** if you want to report a bug or a feature. Don't open issues for questions or support, instead join our [AsyncAPI Slack workspace](https://www.asyncapi.com/slack-invite) and ask there. Don't forget to follow our [Slack Etiquette](https://github.com/asyncapi/community/blob/master/slack-etiquette.md) while interacting with community members! It's more likely you'll get help, and much faster!
 
-## Bug Reports and Feature Requests
+### Bug Reports and Feature Requests
 
 Please use our issues templates that provide you with hints on what information we need from you to help you out.
 
-## Pull Requests
+### Pull Requests
 
-**Please, make sure you open an issue before starting with a Pull Request, unless it's a typo or a really obvious error.** Pull requests are the best way to propose changes to the specification. Get familiar with our document that explains [Git workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) used in our repositories.
+**Please make sure you open an issue before starting with a Pull Request unless it's a typo or a really obvious error.** Pull requests are the best way to propose changes to the specification. If no issue was created first to discuss the need for a pull request, it may be rejected.
 
-## Conventional commits
+Get familiar with our document that explains the [Git workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) used in our repositories.
```

Comment:
```suggestion
Following is a document you can use to get understand and get familiar with the [Git workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) used in our repositories.
```

Reply:
@TRohit20 can we re frame this line in some another way 😅

Reply:
yeah, tbh I don't think it is grammatically correct

Reply:
@derberg I think we can skip this comment and merge the PR as it has been a long time since this is pending